build_tag.sh

```diff
@@ -10,10 +10,16 @@ IMAGE_TEST=addon-metadata-operator
 docker build -t ${IMAGE_TEST} -f Dockerfile.ci .
 docker_run_args=(
     --rm
-    # goreleaser version
-    -e "VERSION=v0.184.0"
+    --privileged
+    -e CGO_ENABLED=1
     # github API token to post release
     -e "GITHUB_TOKEN=${GITHUB_TOKEN}"
+    -v /var/run/docker.sock:/var/run/docker.sock
+    -v $(pwd):/go/src/github.com/mt-sre/addon-metadata-operator
+    -w /go/src/github.com/mt-sre/addon-metadata-operator
+    # goreleaser version
+    goreleaser/goreleaser-cross:v1.17.6
+    release --rm-dist
     --entrypoint=/bin/sh
 )
 docker run "${docker_run_args[@]}" "${IMAGE_TEST}" -c /tmp/goreleaser.sh
```

Comment:
```suggestion
docker run "${docker_run_args[@]}"
```

Reply:
^^ I think Jenkins will fail as it doesnt allow pulling images from dockerhub --> goreleaser/goreleaser-cross:v1.17.6